plugins/router/arris-sb6183

```diff
@@ -80,8 +80,12 @@ GPLv2
 import re
 import os
 import sys
+import ctypes
 from urllib import request
 
+def fixint(i):
```

Comment:
Please move the new function below the "constants".

And please add a short description, what exactly this function "fixes".